sip-ri/src/main/java/gov/nist/javax/sip/header/ParameterNames.java

```diff
@@ -167,4 +167,12 @@ public interface ParameterNames {
     public static final String LOCAL_TAG = "local-tag";
     public static final String REMOTE_TAG = "remote-tag";
 
+    //Support for RFC 5806 @MukhaValeriia
+	public static final String REASON = "reason";
+	public static final String LIMIT = "limit";
+	public static final String COUNTER = "counter";
+	public static final String PRIVACY = "privacy";
+	public static final String SCREEN = "screen";
+	public static final String EXTENSION = "extension";
```

Comment:
This one doesn't exist in the spec, it's a token so a regular param, see the syntax at https://datatracker.ietf.org/doc/html/rfc5806#section-4

diversion-extension = token ["=" (token | quoted-string)]

Reply:
Do we need to check in general the extension of the header? Should I add a check to the test and in parser?

Reply:
given diversion-extension = token ["=" (token | quoted-string)] you can check in the tests that additional random parameters provided by the application can be still parsed and included such as ;random-param=random-value by example